http/headers/feature-policy.json

```diff
@@ -240,10 +254,24 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": false
+                "version_added": "65",
+                "flags": [
```

Comment:
Do we want to have the flag data on all these subfeatures too, if the top-level feature is behind a flag? It's been a while since I've seen something like this, but I'm thinking this is analogous to a CSS property behind a flag, where all its value subfeatures wouldn't duplicate the flag data.

Reply:
Great question! I'm honestly quite unsure about it. For now, I was thinking in practical terms for the displaying MDN pages like https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy/camera which need to show that it isn't available just by default. I'm not sure about data consumers/displayers generally deal with this kind of "data inheritance".

Reply:
Also, as you see in the Chrome data, this might get on by default on the top-level, but then not all sub features get on by default as well at the same time. Tricky.

Reply:
That's a good point about not making consumers deal with inheritance! Let's go with this as written. Thanks, @Elchi3!